assets/styles.css

```diff
@@ -1,3 +1,409 @@
+  font-size: 0.8125rem;
+  font-family: nyt-imperial, georgia, "times new roman", times, serif;
+}
+.big-band {
+  width: 600px;
+  margin: 0 auto;
+}
+nav {
+  margin: 0 auto;
+  width: 1200px;
+  padding-bottom: 20px;
+  padding-top: 9px;
+  border-bottom: 1px solid #e2e2e2;
+  border-top: 2px solid #e2e2e2;
+  content: "";
+  display: block;
+}
+.nav-header {
+  display: flex;
+  flex-flow: row;
+  height: 45px;
+}
+.go-to-home {
+  margin: 18px 0px 0px auto;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-size: 11px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 11px;
+}
+.go-to-home a {
+  color: rgb(86, 123, 149);
+  text-decoration: none;
+}
+.nav-sections {
+  display: flex;
+  flex-flow: row;
+  padding-right: 20px;
+  /* text-decoration: none; */
+  color: #333;
+  width: 1200px;
+  margin: 0 auto;
+}
+.nav-section-column {
+  width: 20%;
+}
+.nav-section-column h3 {
+  color: rgb(51, 51, 51);
+  font-size: 13px;
+  font-weight: 700;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  height: 25px;
+  line-height: 15px;
+  margin: 0px;
+  text-transform: uppercase;
+  width: 150px;
+}
+.nav-section-column ul {
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+  font: inherit;
+}
+.nav-section-column li a {
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+  color: rgb(0, 0, 0);
+  display: inline-block;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  text-decoration: none;
+  text-transform: capitalize;
+  width: 150px;
+  font-size: 14px;
+  font-weight: 500;
+  height: 23px;
+  line-height: 16px;
+}
+.subscribe {
+  border-left: 1px solid rgb(226, 226, 226);
+  /* flex: 1 1 0%; */
+  padding-left: 15px;
+}
+.subscribe-one {
+  margin: 0px;
+  padding: 0px;
+  margin-bottom: 5px;
+  white-space: nowrap;
+}
+.subscribe-one a {
+  color: rgb(0, 0, 0);
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  text-decoration: none;
+  text-transform: capitalize;
+  width: 150px;
+  font-size: 16px;
+  font-weight: 700;
+  height: 25px;
+  line-height: 15px;
+  padding-bottom: 0px;
+}
+
+.subscribe-one svg {
+  display: inline-block;
+  height: 13px;
+  width: 13px;
+  margin-right: 2px;
+  vertical-align: middle;
+}
+.subscribe-two {
+  margin: 0px;
+  padding: 0px;
+  margin-bottom: 5px;
+  white-space: nowrap;
+  font-size: 11px;
+}
+.subscribe-two a {
+  color: rgb(0, 0, 0);
+  display: inline-block;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  text-decoration: none;
+  text-transform: capitalize;
+  width: 150px;
+  font-size: 11px;
+  font-weight: 500;
+  height: 23px;
+  line-height: 21px;
+  cursor: pointer;
+  text-decoration: underline;
+}
+.subscribe-three {
+  list-style: none;
+  margin: 2px 0px 0px;
+  padding: 10px 0px 0px;
+  border-top: 1px solid rgb(226, 226, 226);
+}
+.end-article-block {
+  width: 600px;
+  margin: 0 auto;
+}
+.end-article-text {
+  /* font-family: nyt-franklin, helvetica, arial, sans-serif; */
```

Comment:
- [ ]  Commented code is not a good practice in clean coding. Please avoid it.

Reply:
Done.